### PR TITLE
Order: Base features

### PR DESCRIPTION
**Pull request by xzzz3**
Creating this new PR to replace #74 #75 #76 as a consolidated PR that merges the three together. Closes #76

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditOrderStatusCommand.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+        Order orderToEdit = lastShownList.get(index.getZeroBased());
+        Order editedOrder = orderToEdit.updateStatus(status);
+
+        if (!editedOrder.equals(orderToEdit) && model.hasOrder(editedOrder)) {
+            throw new CommandException(MESSAGE_DUPLICATE_ORDER);
+        }
+
+        model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
+        return new CommandResult(String.format(MESSAGE_SUCCESS, editedOrder));
```
Comment (hieunm1821) [inline src/main/java/seedu/address/logic/commands/EditOrderStatusCommand.java:60]: I think the constructor of CommandResult have changed to fit the new UI. 
`CommandResult(String feedbackToUser, boolean showHelp,
            boolean exit, boolean isDish, boolean isDriver, boolean isOrder)`
I think same issues with other types of Command too.

File: src/main/java/seedu/address/logic/parser/EditOrderStatusCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+<<<<<<< HEAD
+ * Parses input arguments and creates a new EditOrderCommand object
+=======
+ * Parses input arguments and creates a new EditOrderStatusCommand object
+>>>>>>> Feature-FindOrderByPhone
+ */
+public class EditOrderStatusCommandParser {
+
+    /**
+<<<<<<< HEAD
```
Comment (hieunm1821) [inline src/main/java/seedu/address/logic/parser/EditOrderStatusCommandParser.java:22]: There are some merging issue have not solved yet.

File: src/main/java/seedu/address/model/order/UniqueOrderList.java [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+        // todo implement throw exception for duplicate
+        internalList.add(toAdd);
+    }
+
+    /**
+     * Replaces the order {@code target} in the list with {@code editedorder}.
+     * {@code target} must exist in the list.
+     * The order identity of {@code editedorder} must not be the same as another existing order in the list.
+     */
+    public void setorder(Order target, Order editedOrder) {
```
Comment (hieunm1821) [inline src/main/java/seedu/address/model/order/UniqueOrderList.java:52]: The method name should be `setOrder`

File: src/main/java/seedu/address/model/order/UniqueOrderList.java [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+     * The order must exist in the list.
+     */
+    public void remove(Order toRemove) {
+        requireNonNull(toRemove);
+        if (!internalList.remove(toRemove)) {
+            throw new PersonNotFoundException(); // todo replace exception
+        }
+    }
+
+    public void setorders(seedu.address.model.order.UniqueOrderList replacement) {
```
Comment (hieunm1821) [inline src/main/java/seedu/address/model/order/UniqueOrderList.java:78]: The method name should be setOrders

**Comment by xzzz3** [comment]
@hieunm1821 thanks for the reviews! I have fixed them and added some new elements 😺

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 5]
```diff
@@ -196,6 +196,15 @@ private void handleDriver() {
         logger.info("Set to driver");
     }
 
+    /**
+     * Handles case when order command is entered.
+     */
+    private void handleOrder() {
+        listPanel = new OrderListPanel(logic.getFilteredOrderList());
+        personListPanelPlaceholder.getChildren().add(listPanel.getRoot());
+        tabDisplay.setFocus(DISH_TAB_DISPLAY_FOCUS);
```
Comment (hieunm1821) [inline src/main/java/seedu/address/ui/MainWindow.java:205]: Just small issue. It should be `ORDER_TAB_DISPLAY_FOCUS`

**Review by zxgoh — COMMENTED** [review]
Looks okay to me, I will fix conflicts when this merges

**Review by zxgoh — APPROVED** [review]

File: src/main/java/seedu/address/logic/commands/AddOrderCommand.java [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+public class AddOrderCommand extends Command {
+
+    public static final String COMMAND_WORD = "addorder";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Adds an order.\n"
+            + "Parameters: "
+            + "p/ [PHONE] d/ [DISHES]\n"
+            + "Example: " + COMMAND_WORD
+            + "p/ 87654321 d/ Chicken Pasta";
```
[the remark this answers is not in the document]
Comment (xzzz3) [reply src/main/java/seedu/address/logic/commands/AddOrderCommand.java]: ok, fixed that!

**Comment by xzzz3** [comment]
Merging this PR with the main PR. resolves issues #33 , #34 , #51 .

**Pull request merged**